### DISCUSSION TITLE
Allow the formatUsing method to output HTML tags

### DISCUSSION THREAD

**Discussion by rawzone** [filamentphp/filament#96] [Help] [answered]
When using the formatUsing method for setting up index table columns it could be great if allowed to output HTML tags.

By allowing this you could super easy output a `<img src="' . $value . '">` tag or lets say a `<div class="w-6 h-6 rounded-full" style="background-color:' . $value . ';"></div>` to show a colored circle in the tables. (If a model saves a color code with a text input).

**Comment by rawzone** [comment]
In context to that it would also be nice to have the ->sortable method allowed to have a "sort value" passed to it so it could allow for custom sorting of data undepended on the output.

**Comment by danharrin** [comment, marked as answer]
Hey! I'm not going to implement this, as it would open Filament up to potential XSS attacks.

You should instead use the `->view()` method on your column to specify the path to a custom view which renders `$column->getValue($record)`.

**Reply by rawzone** [reply]
Thats even better! Dident see that in the docs.
Works like a charm!
